include/examples/simple_limb.h

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file simple_limb.h
+ * @author Kodlab - Zac Gong (zacgong@seas.upenn.edu)
+ * @author Kodlab - J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @brief A simple LimbBase derived class example, a 1DOF limb with 1 joint at
+ *        the origin and a end effector
+ * @date 2022-09-02
+ *
+ * @copyright (c) Copyright 2022 The Trustees of the University of Pennsylvania
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Dense>
+
+#include "kodlab_mjbots_sdk/limb_base.h"
+
+/**
+ * @brief A simple 1-DOF limb consisting of a single joint with a striaght
+ * link to an end-effector in Cartesian XYZ space
+ * @details A simple, single-degree-of-freedom limb implementation demonstrating 
+ * the specification of a Cartesian offset and implementation of kinematics
```

Comment:
Shouldn't mention cartesian offset since we are not specifying what that is.

Reply:
It is specific to the config for this file, I'll check that this is correct before merging